controllers/controller.js

```diff
@@ -24,4 +25,19 @@ const {
     .catch(err => {
       next(err);
     });
-  }
\ No newline at end of file
+  }
+
+  exports.getArticleById = (req, res, next) => {
+   
+    const { article_id } = req.params;
+    selectArticleById(article_id).then((article) => {
+      if (article === "No article") {
+       return next("No match found");
+      }
```

Comment:
Why would your model return a string of `"No article"`?

This looks like it's not needed to me as the model will reject if there is no article.